tests/subsys/audio_module/src/bad_param_test.c

```diff
@@ -0,0 +1,1172 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <zephyr/ztest.h>
+#include "fakes.h"
+#include <errno.h>
+
+#include "audio_module/audio_module.h"
+#include "audio_module_test_common.h"
+
+static struct audio_module_functions mod_1_functions = {.open = NULL,
+							.close = NULL,
+							.configuration_set = NULL,
+							.configuration_get = NULL,
+							.start = NULL,
+							.stop = NULL,
+							.data_process = NULL};
+
+static struct audio_module_description test_description, test_description_1, test_description_2,
+	test_description_tx, test_description_rx;
+static struct mod_config configuration;
+static struct audio_module_configuration *config =
+	(struct audio_module_configuration *)&configuration;
+static struct audio_module_handle handle, handle_tx, handle_rx;
+static struct mod_context context;
+static uint8_t test_data[FAKE_FIFO_MSG_QUEUE_DATA_SIZE];
+static struct data_fifo mod_fifo_tx, mod_fifo_rx;
+static struct audio_data test_block, test_block_tx, test_block_rx;
+static char mod_thread_stack[TEST_MOD_THREAD_STACK_SIZE];
+
+static void test_initialise_description(struct audio_module_description *description, char *name,
```

Comment:
```suggestion
static void test_initialize_description(struct audio_module_description *description, char *name,
```
Same for test_initialise_handle. Should be sufficient to just do a search and replace all in this file